src/components/ha-clickable-list-item.ts

```diff
@@ -0,0 +1,66 @@
+import { style } from "@material/mwc-list/mwc-list-item-css";
+import { ListItemBase } from "@material/mwc-list/mwc-list-item-base";
+import { css, CSSResult, customElement, property } from "lit-element";
+import { html } from "lit-html";
+
+@customElement("ha-clickable-list-item")
+export class HaClickableListItem extends ListItemBase {
```

Comment:
Originally was extending `ListItem` (which adds styles). But that through a typescript error when later overriding `styles` to be `CSSResult[]` instead of `CSSResult`, which I did so I wouldn't have to manually import the material styling myself.

The correct way to do this is to override `ListItemBase` and pull style in yourself. See https://github.com/material-components/material-components-web-components/issues/322.